src/shared/utils/unsupportedExtensionsMapper.ts

```diff
@@ -348,9 +348,9 @@ const imageFileExtensions = new Set([
   'xpm',
 ])
 
-export function unsupportedExtensionsMapper({ path }) {
-  const fileName = getFilenameFromPath(path)
-  const fileExtension = getFileExtension(fileName)
+export function unsupportedExtensionsMapper({ path }: { path: string }) {
+  const fileName = getFilenameFromPath(path) ?? ''
```

Comment:
This is the main fix, just default to an empty string if doesn't exist in either of these first two functions